testcode/tinygraphsolver.cpp

```diff
@@ -133,22 +137,21 @@ int main(int argc, char* argv[])
     mfem::Array<int>& blockOffsets(mixed_graph_laplacian.get_blockoffsets());
     mfem::BlockVector rhs = *mixed_graph_laplacian.subvecs_to_blockvector(rhs_u_fine, rhs_p_fine);
     mfem::BlockVector sol(blockOffsets);
-    sol = 0.0;
 
-    // solve
-    MinresBlockSolver mgp(comm, mixed_graph_laplacian);
-    mgp.SetPrintLevel(1);
-    mgp.Mult(rhs, sol);
+    // setup solvers
+    std::map<const MixedLaplacianSolver*, std::string> solver_to_name;
```

Comment:
@barker29 @gelever Here I am using the setting in `poweriter.cpp`. this file should contain the main new modifications, can you guys please take another look. After that I guess I will merge this branch.

Reply:
Looks good to me.